owlbot.py

```diff
@@ -14,4 +14,4 @@
 """This script is used to synthesize generated parts of this library."""
 import synthtool.languages.node as node
 
-node.owlbot_main(staging_excludes=['package.json', 'README.md', 'CODEOWNERS'])
+node.owlbot_main(staging_excludes=['package.json', 'README.md', '.github/CODEOWNERS'])
```

Comment:
What was the change needed to CODEOWNERs to support this?  I really want to avoid excluding files here as a habit if we can.

Reply:
Totally! Right now this is just an experiment, because I'm testing auto-approve bot on this library. In the future, I'd love to have the GH Automation line for auto-approve be a part of the template we ship out to our libraries :)